GNUmakefile

```diff
@@ -280,6 +282,22 @@ pool-mgmt-image: cstor-pool-mgmt
 	@cd buildscripts/cstor-pool-mgmt && sudo docker build -t ${HUB_USER}/${CSTOR_POOL_MGMT_REPO_NAME}:${IMAGE_TAG} --build-arg BASE_IMAGE=${CSTOR_BASE_IMAGE} --build-arg BUILD_DATE=${BUILD_DATE} . --no-cache
 	@rm buildscripts/cstor-pool-mgmt/${POOL_MGMT}
 
+#Use this to build cspi-mgmt
+cspi-mgmt:
+	@echo "----------------------------"
+	@echo "--> cspi-mgmt           "
+	@echo "----------------------------"
+	@PNAME="cspi-mgmt" CTLNAME=${CSPI_MGMT} sh -c "'$(PWD)/buildscripts/build.sh'"
```

Comment:
Let's set PNAME to `${CSPI_MGMT}`.

Reply:
In buildscripts/build we are setting `main.CtlName` during linking for each cmd under `openebs/maya/cmd/`, but we don't have any variable named `CtlName`.